### PR TITLE
Issue 81 - Settings pane 

### PR DESCRIPTION
**Pull request by RaviRajput10**
Fixes: #81

### CODE REVIEW

File: apps/web/app/dashboard/app/[name]/appLinks.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
+"use client";
```
Comment (rajat1saxena) [inline apps/web/app/dashboard/app/[name]/appLinks.tsx:1]: Use consistent file naming patterns across the project. Change it to `app-links.tsx`.

File: apps/web/app/dashboard/app-operations.tsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
+"use client";
```
Comment (rajat1saxena) [inline apps/web/app/dashboard/app-operations.tsx:1]: Change the name to `new-app-button.tsx`

File: apps/web/app/dashboard/app/[name]/appLinks.tsx (hunk 1 above)
Comment (rajat1saxena) [reply apps/web/app/dashboard/app/[name]/appLinks.tsx:1]: Change the file name to `tabs.tsx`.

Comment (RaviRajput10) [reply apps/web/app/dashboard/app/[name]/appLinks.tsx]: Changed the file name

File: apps/web/app/dashboard/app-operations.tsx (hunk 2 above)
Comment (RaviRajput10) [reply apps/web/app/dashboard/app-operations.tsx]: Changed

**Review by rajat1saxena — APPROVED** [review]

**Pull request merged**